source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/MarkDownUnitTest.cpp

```diff
@@ -249,13 +249,13 @@ namespace AdaptiveCardsSharedModelUnitTest
         TEST_METHOD(Rule11_12Test_EscapeTest)
         {
             MarkDownParser parser("foo *\\**");
-            Assert::AreEqual<std::string>("<p>foo <em>*</em></p>", parser.TransformToHtml());
+            Assert::AreEqual<std::string>("<p>foo <em>\\</em>*</p>", parser.TransformToHtml());
 
             MarkDownParser parser2("foo **\\***");
-            Assert::AreEqual<std::string>("<p>foo <strong>*</strong></p>", parser2.TransformToHtml());
+            Assert::AreEqual<std::string>("<p>foo <strong>\\</strong>*</p>", parser2.TransformToHtml());
 
             MarkDownParser parser3("foo __\\___");
-            Assert::AreEqual<std::string>("<p>foo <strong>_</strong></p>", parser3.TransformToHtml());
```

Comment:
I used Commonmark's editor to generate the correct output
```
<p>foo <em>\</em>*
foo <strong>\</strong>*
foo <strong>\</strong>_</p>
```
escaped chars within the double quotes are single characters. so \\\\ within double quotes is a single \\ in memory and outputted as shown in the above example.